cmd/skaffold/app/cmd/config/flags.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2018 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var configFile, kubectx string
+
+func AddConfigFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVarP(&configFile, "config", "c", "", "path to skaffold config")
+	cmd.Flags().StringVarP(&kubectx, "kubectx", "k", "", "kubectl context to set values against")
```

Comment:
Please don't use term "kubectx". It's a third-party project name https://github.com/ahmetb/kubectx and is not widely recognized in the context of kubectl. Be verbose and accurate, so I recommend `--kubectl-context` here instead.